src/Language/id/ShieldOAuthLang.php

```diff
@@ -14,11 +14,11 @@
 // ShieldOAuthLang language file
 return [
     // Errors List
-    'unknown'  => 'Ups, kesalahan tidak terduga!',
+    'unknown'  => 'Kesalahan tidak diketahui!',
     'Callback' => [
-        'oauth_class_not_set' => 'Terjadi kesalahan, sepertinya class OAuth tidak terpasang.',
-        'anti_forgery'        => 'Maaf, permintaan Anda terdeteksi tidak valid!',
-        'account_not_found'   => 'Akun tidak ditemukan, silakan mendaftar terlebih dahulu!',
+        'oauth_class_not_set' => 'Terjadi Kesalahan: sepertinya class "OAuth" tidak terpasang.',
+        'anti_forgery'        => 'Permintaan ini terdeteksi tidak valid!',
```

Comment:
One thing at a time: A pull request should only contain one change. That does not mean only one commit, but one change - however many commits it took. The reason for this is that if you change X and Y, but send a pull request for both at the same time, we might really want X but disagree with Y, meaning we cannot merge the request. Using the Git-Flow branching model you can create new branches for both of these features and send two requests.

So if you think translation **id** needs improvement, separate it in a PR.